recipes/ganon/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "ganon" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/pirovc/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 84814caf51616911166840819fd14b9436bcf2cdfd261040ac1064a9acd4d16a
```

Comment:
You can use here multiple URLs and checksums to download multiple repos. This would remove the git dependency and we can backup the tarball.

Reply:
thanks, it makes the recipe much better indeed